### PR TITLE
docs(core): mark TestBed as stable api and add preliminary docs

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

File: modules/@angular/core/testing/test_bed.ts [hunk 1]
```diff
@@ -47,7 +47,12 @@ export type TestModuleMetadata = {
 };
 
 /**
- * @experimental
+ * @whatItDoes Configures and initializes environment for unit testing and provides methods for creating components and services in unit tests.
```
Comment (vicb) [inline modules/@angular/core/testing/test_bed.ts]: line length

**Review by vicb — CHANGES_REQUESTED** [review]
fix the CI please

**Comment by IgorMinar** [comment]
done

File: modules/@angular/core/testing/test_bed.ts [hunk 2]
```diff
@@ -47,7 +47,13 @@ export type TestModuleMetadata = {
...
 
 /**
- * @experimental
+ * @whatItDoes Configures and initializes environment for unit testing and provides methods for
+ * creating components and services in unit tests.
+ * @description
+ *
+ * TestBed is the primary api for writing unit tests for Angular applications and libraries.
+ *
+ * @stable
```
Comment (vicb) [inline modules/@angular/core/testing/test_bed.ts]: please update the public api

**Review by vicb — CHANGES_REQUESTED** [review]

File: modules/@angular/core/testing/test_bed.ts (hunk 2 above)
Comment (IgorMinar) [inline modules/@angular/core/testing/test_bed.ts]: duh. I did, but incorrectly. hopefully it's fixed now.

**Review by vicb — APPROVED** [review]

**Pull request merged**